GPU/GPUCommonHW.cpp

```diff
@@ -822,7 +822,7 @@ void GPUCommonHW::FastRunLoop(DisplayList &list) {
 void GPUCommonHW::Execute_VertexType(u32 op, u32 diff) {
 	if (diff)
 		gstate_c.Dirty(DIRTY_VERTEXSHADER_STATE);
-	if (diff & (GE_VTYPE_TC_MASK | GE_VTYPE_THROUGH_MASK)) {
+	if (diff & (GE_VTYPE_THROUGH_MASK)) {
 		gstate_c.Dirty(DIRTY_UVSCALEOFFSET);
```

Comment:
I think we can remove this, and only have the inner if.

-[Unknown]

Reply:
Actually, we should also make this function smarter and avoid flushing in many cases, just like Execute_VertexTypeSkinning. We still can use these improvements even if software skinning is off.